src/components/player/ScoreEntry.tsx

```diff
@@ -0,0 +1,24 @@
+function ScoreEntry({
+  player,
+  currentPlayer,
+  playerScore,
+}: {
+  player: [string, number];
+  currentPlayer: string;
```

Comment:
instead of passing in `currentPlayer` and performing the checks that rely on the `currentPlayer` value, you should pass in the result of those checks and use them accordingly. ex:
```js
<ScoreEntry
    isCurrPlayer={player[0] === currentPlayer}
    playerName={player[0]}
    playerScore={player[1]} 
/>
```